osquery/tables/events/windows/windows_events.cpp

```diff
@@ -61,14 +61,24 @@ REGISTER(WindowsEventSubscriber, "event_subscriber", "windows_events");
 /// Helper function to recursively parse a boost ptree
 void parseTree(const pt::ptree& tree, std::map<std::string, std::string>& res) {
   for (const auto& node : tree) {
-    auto nodeName = node.second.get("<xmlattr>.Name", "");
+    // Skip this since it's not actually part of the EventData. Also prevents
+    // us from adding every Name attribute into its own key invalidly. This is
+    // part of a quirk of boost::ptree and its parsing of XML.
+    if (node.first == "<xmlattr>") {
+      continue;
+    }
 
+    auto nodeName = node.second.get("<xmlattr>.Name", "");
```

Comment:
Prefer snake case for variables

Reply:
This is definitely the way it _should_ be done, however the entirety of the windows code base is currently incorrect :( I'd say for now stick with consistency and we can file a follow-up item to fix everything.

Reply:
Ya - I was imitating most of the other functions in the file but I did note that there's both snake case and pascal case floating around.